torch/utils/_pytree.py

```diff
@@ -336,9 +335,8 @@ def go(t):
 
     You can also directly use 'tree_map_only'
     """
-    def deco(f: Callable[[T], Any]) -> Callable[[Any], Any]:
-        @functools.wraps(f)
-        def inner(x: T) -> Any:
+    def deco(f: Any) -> Any:
+        def inner(x: Any) -> Any:
```

Comment:
because the type annotations are not supported?

Reply:
As far as I remember, in order to support type annotations, I had to implement `UserDefinedObjectVariable.as_python_constant`. That lead to some CI failures, of which I don't remember, right now. I will try to revert this and see what happens.

Reply:
Nevermind. I just had to make `Any` a `TypingVariable`.